package.json

```diff
@@ -47,7 +47,8 @@
     "ts-generator": "^0.1.1",
     "ts-node": "^10.2.1",
     "typechain": "^5.1.2",
-    "typescript": "^4.4.3"
+    "typescript": "^4.4.3",
+    "solc": "0.8.4"
```

Comment:
Why this needs to be included?

Reply:
I noticed that there was no Solidity compiler installed locally for the project. Depending on your development environment setup, it can be configured to either use a globally installed Solidity compiler or a locally installed one for different tasks like linting or compiling. 
Specifying the exact Solidity compiler package allows for easier reproducible builds as it guarantees that there are no clashes with the necessary Solc and the systems installed Solc.

Reply:
Hardhat installs correct solidity compiler automatically.